class.jetpack-jitm.php

```diff
@@ -21,12 +21,53 @@ static function init() {
 	}
 
 	private function __construct() {
+		if ( ! Jetpack::is_active() ) {
```

Comment:
while we call the `current_user_can()` function after `capabilities.php` has loaded `wp_get_current_user()` is not yet available, so this will result in an error. There may be a way to hook into it if we require the file